### PR TITLE
modify one log of eventbus

### PR DESCRIPTION
**Pull request by muxuelan**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
For version 1.4，one log of the function onSubscribe is unreasonable.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/eventbus/mqtt/client.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func OnSubMessageReceived(client MQTT.Client, message MQTT.Message) {
 	klog.Infof("OnSubMessageReceived receive msg from topic: %s", message.Topic())
 	// for "$hw/events/device/+/twin/+", "$hw/events/node/+/membership/get", send to twin
 	// for other, send to hub
-	// for topic, no need to base64 topic
+	// for "SYS/dis/upload_records", no need to base64 topic
```
Comment (daixiang0) [inline edge/pkg/eventbus/mqtt/client.go:97]: I think it is still a topic name. Maybe better to comment in doc, and try to add unit test fort this :)

Comment (muxuelan) [reply edge/pkg/eventbus/mqtt/client.go:97]: Actually，all topis are 64-encoded，except for "SYS/dis/upload_records".I think  the sentence "for topic"range too much and is incorrect here.

Comment (daixiang0) [reply edge/pkg/eventbus/mqtt/client.go:97]: OK.

**Pull request merged**